Cargo.toml

```
@@ -1425,6 +1425,16 @@ description = "Shows a visualization of gamepad buttons, sticks, and triggers"
 category = "Tools"
 wasm = false
 
+[[example]]
+name = "internal_ambiguities"
+path = "examples/tools/internal_ambiguities.rs"
+
+[package.metadata.example.internal_ambiguities]
+name = "Internal Ambiguities"
+description = "Verify that `DefaultPlugins` works, and that the schedule is free of problematic execution order ambiguities."
+category = "Tools"
+wasm = false
```

With this being intended as a regression test (and not as an example that you want to prevent from having regressions), I think this should be moved to the [`./tests`](https://github.com/bevyengine/bevy/tree/main/tests) folder.

```suggestion
[[example]]
name = "internal_ambiguities"
path = "tests/internal_ambiguities.rs"

[package.metadata.example.internal_ambiguities]
hidden = true
```

---

Hmm. I like that suggestion. I'm not sure that will work though; CI will try to run all of the tests in that folder and this example needs a GPU to run.

---

:thinking: That complicates this.

I know unit- and doc-tests can be marked as `ignored`. With this, you could skip the test in the regular CI, but still, run it separately in a CI Job with Software Renderer.

The problem is that I can't find anything about whether integration tests can be marked as `ignored`.

If no other Solution is found, then I am with the PR as is.

P.S. How do the existing Tests in `./tests/window/` not fail the CI, even though both use `DefaultPlugins` aka need a GPU?

---

Yeah I'll try this out and see what CI does!